fidap/fidap.py

```diff
@@ -36,6 +37,19 @@ def sql(self, sql, db=None):
         if db:
             self._custom_db = db
         return self.api({'sql_query': sql, **self.api_keys})
+    
+    def load_table_as_pandas(self, share_name = "fidap_share", schema_name = None, table_name= None):
+        file_path = "https://fidap.s3-us-west-2.amazonaws.com/fidap_data.share"
```

Comment:
This url seems to be configuration so it should be in class variable then we can access it easily, instead of copy and paste it everywhere.